packages/compartment-mapper/test/policy-format.test.js

```diff
@@ -62,6 +62,12 @@ const q = JSON.stringify;
   {
     globals: ['a', {}],
   },
+  {
+    options: {
```

Comment:
Do we care that it's possible to put unserializable junk in here?  Should that be restricted via types or at runtime or both or neither?

Reply:
I defer to @naugtur